libbeat/idxmgmt/ilm/client_handler.go

```diff
@@ -60,24 +58,17 @@ type FileClientHandler struct {
 }
 
 // FileClient defines the minimal interface required for the Loader to
-// prepare a policy and write alias.
+// prepare a policy.
 type FileClient interface {
 	GetVersion() common.Version
 	Write(component string, name string, body string) error
 }
 
 const (
-	// esFeaturesPath is used to query Elasticsearch for availability of licensed
-	// features.
-	esFeaturesPath = "/_xpack"
-
 	esILMPath = "/_ilm/policy"
-
-	esAliasPath = "/_alias"
 )
 
 var (
-	esMinILMVersion        = common.MustNewVersion("6.6.0")
 	esMinDefaultILMVersion = common.MustNewVersion("7.0.0")
```

Comment:
Do we still need this as only 8.0.0 is supported anyways?

Reply:
No, we don't. But after a while, I tried to minimize the amount of clean up in the PR because it got out of hand. I will open a follow-up PR for more clean ups.